### ISSUE TITLE
Probrem with zeitwerk

### ISSUE THREAD

**Issue by alec-c4** [kirillplatonov/hotwire-livereload#26]
```
Loading application environment...
Loading code in search of Active Record models...
rails aborted!
Loading models failed!
Error occurred while loading application: expected file /Users/alec/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/hotwire-livereload-1.2.0/app/channels/hotwire/livereload/reload_channel.rb to define constant Hotwire::Livereload::ReloadChannel, but didn't

      raise Zeitwerk::NameError.new("expected file #{file} to define constant #{cpath}, but didn't", cref.last)
      ^^^^^ (Zeitwerk::NameError)


Caused by:
Zeitwerk::NameError: expected file /Users/alec/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/hotwire-livereload-1.2.0/app/channels/hotwire/livereload/reload_channel.rb to define constant Hotwire::Livereload::ReloadChannel, but didn't

      raise Zeitwerk::NameError.new("expected file #{file} to define constant #{cpath}, but didn't", cref.last)
      ^^^^^

Tasks: TOP => erd => erd:generate => erd:load_models
(See full trace by running task with --trace)
```

Hi, looks like a problem with zeitwerk. To reproduce - just create an app with template https://github.com/alec-c4/ks-rails-bootstrap then run `bundle exec rails g erd:install` and `bundle exec rails db:migrate`.

**Comment by kirillplatonov** [comment]
Hey, 

This problem seems to be caused by a return statement at the beginning of `reload_channel.rb`: [link](https://github.com/kirillplatonov/hotwire-livereload/blob/main/app/channels/hotwire/livereload/reload_channel.rb#L1). I suppose we could just remove it.

**Comment by alec-c4** [comment]
Thanks :)

**Comment by kirillplatonov** [comment]
Published `v1.2.1` with a fix.
